ckanext/benap/benap_schema.json

```diff
@@ -422,6 +423,27 @@
       "benap_helper_ontology": "NUTS1_BE",
       "required": "true"
     },
+    {
+      "field_name": "network_coverage",
+      "label": {
+        "en": "Network Coverage",
+        "fr": "Couveerture réseau",
```

Comment:
In het Engels doe je Network Coverage (beide woorden met hoofdletter). In het Frans Couveerture réseau (2e woord dus kleine letter). Misschien hier consistent in zijn?